src/main/asciidoc/webapp/webapp001.adoc

```diff
@@ -1,46 +1,49 @@
-The web server converts this object to an HTTP response and returns it to the client.
+Development of service-oriented web applications is covered in [TBD]
+
+In the Jakarta EE platform, web applications are represented by web components as seen in xref:jakarta-web-application-request-handling[xrefstyle=short].
+A web component can be represented by Jakarta Servlet, Jakarta Faces or Jakarta REST.
 
 [[jakarta-web-application-request-handling]]
 .Jakarta Web Application Request Handling
 image::jakartaeett_dt_013.svg["Diagram of web application request handling. Clients and servlets communicate using HttpServletRequest and HttpServletResponse."]
 
-Servlets are Java programming language classes that dynamically process requests and construct responses.
-Java technologies, such as Jakarta Faces and Facelets, are used for building interactive web applications.
-(Frameworks can also be used for this purpose.)
-Although servlets and Jakarta Faces and Facelets pages can be used to accomplish similar things, each has its own strengths.
-Servlets are best suited for service-oriented applications (web service endpoints can be implemented as servlets) and the control functions of a presentation-oriented application, such as dispatching requests and handling nontextual data.
-Jakarta Faces and Facelets pages are more appropriate for generating text-based markup, such as XHTML, and are generally used for presentation-oriented applications.
-
-Web components are supported by the services of a runtime platform called a web container.
-A web container provides such services as request dispatching, security, concurrency, and lifecycle management.
-A web container also gives web components access to such APIs as naming, transactions, and email.
-
-Certain aspects of web application behavior can be configured when the application is installed, or deployed, to the web container.
-The configuration information can be specified using Jakarta EE annotations or can be maintained in a text file in XML format called a web application deployment descriptor (DD).
-A web application DD must conform to the schema described in the Jakarta Servlet specification.
-
-This chapter gives a brief overview of the activities involved in developing web applications.
-First, it summarizes the web application lifecycle and explains how to package and deploy very simple web applications on GlassFish Server.
-The chapter then moves on to configuring web applications and discusses how to specify the most commonly used configuration parameters.
+. The client sends an HTTP request to the web server.
+. The web server module that supports Jakarta Servlet based web components is called a servlet container.
+. The servlet container converts the HTTP request into an `HttpServletRequest` object and prepares the `HttpServletResponse` object.
+. These objects are delivered to a web component, which can interact with JavaBeans components or a database to generate dynamic content.
```

Comment:
Personally not a fan of wording "JavaBeans components". How is this term used in CDI chapters?